### PR TITLE
armor: simplify formula.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Also: test to see if the test still hangs after https://github.com/Homebrew/brew/pull/7368

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by Bo98** [comment]
Ordering matters with Go args. The filename should be last.

**Comment by MikeMcQuaid** [comment]
@Bo98 Thanks, updated!

File: Formula/armor.rb [hunk 1]
```diff
@@ -16,19 +16,16 @@ class Armor < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-ldflags", "-s -w", "-trimpath", "-o", bin/"armor", "cmd/armor/main.go"
+    system "go", "build", *std_go_args, "-ldflags", "-s -w", "cmd/armor/main.go",
```
Comment (Bo98) [inline Formula/armor.rb:19]:
```suggestion
    system "go", "build", *std_go_args, "-ldflags", "-s -w", "cmd/armor/main.go"
```

**Comment by MikeMcQuaid** [comment]
@homebrew/core two things worth noting:

- this formula was previously hanging on GitHub Actions so it seems I may have fixed a bunch of hanging tests
- we shouldn't need to have any `Process.kill` cleanup in tests any more. As a result: you can remove these from existing formulae and (until we get a style check) please ensure they aren't added in new formulae. I'd still advise putting these formulae through CI first to ensure they don't hang.

**Pull request merged**